server/index.js

```diff
@@ -37,6 +40,31 @@ app.get('/products/:product_id', (req, res) => {
     });
 });
 
+app.get('/reviews', (req, res) => {
+  const { query } = url.parse(req.url);
```

Comment:
I think you may have added a little complexity here. Query params are available via `req.query` so if you were looking for the product_id, you could get with `req.query.product_id`. I'd rather do that than use URL parsing.

Reply:
I was thinking req.params, but it only for :id, req.query is the good one. I will refactor it.